sdk/bpf/c/inc/solana_sdk.h

```diff
@@ -170,12 +170,14 @@ SOL_FN_PREFIX bool SolPubkey_same(const SolPubkey *one, const SolPubkey *two) {
  */
 typedef struct {
   SolPubkey *key;      /** Public key of the account */
-  bool is_signer;      /** Transaction was signed by this account's key? */
```

Comment:
@mvines Any opposition to this structure re-org, saves 8 bytes :-)

Reply:
nope!  We prolly ought to get this on 1.0.x for TdS